tests/test_10_vasculature.py

```diff
@@ -63,6 +65,34 @@ def test_section_types():
         vasculature.Vasculature(os.path.join(_path, "h5/vasculature-broken-section-type.h5"))
 
 
+def test_section_offsets():
+    morphology = vasculature.Vasculature(os.path.join(_path, "h5/vasculature1.h5"))
+
+    offset = 0
+    expected_offsets = [offset]
+
+    for sec in morphology.sections:
+
+        offset += len(sec.points)
+        expected_offsets.append(offset)
+
+    npt.assert_allclose(morphology.section_offsets, expected_offsets)
```

Comment:
can you add an example of using the `...for convenience,...` with a comment, please?

Reply:
done